config/sync/update.settings.yml

```diff
@@ -0,0 +1,12 @@
+_core:
+  default_config_hash: xbYr66-g0FjNgVBkGypCuN46vBI2XHntXN1URawq1s4
```

Comment:
I believe this configuration is for the update_status module which is only enabled on dev/local. This configuration belongs in those config_split's then and not here.